pkg/controller/realmadmin/save.go

```diff
@@ -44,13 +46,15 @@ func init() {
 
 func (c *Controller) HandleSave() http.Handler {
 	type FormData struct {
-		Name              string            `form:"name"`
-		RegionCode        string            `form:"regionCode"`
-		AllowedTestTypes  database.TestType `form:"allowedTestTypes"`
-		MFAMode           int16             `form:"MFAMode"`
-		EmailVerifiedMode int16             `form:"emailVerifiedMode"`
-		RequireDate       bool              `form:"requireDate"`
-		WelcomeMessage    string            `form:"welcomeMessage"`
+		Name               string            `form:"name"`
+		RegionCode         string            `form:"regionCode"`
+		AllowedTestTypes   database.TestType `form:"allowedTestTypes"`
+		MFAMode            int16             `form:"MFAMode"`
+		EmailVerifiedMode  int16             `form:"emailVerifiedMode"`
+		PasswordRotateDays uint              `form:"pwdRotate"`
```

Comment:
I'd prefer if this was "password" instead of "pwd" because "pwd" can also mean "print working directory" in some contexts.